services/brig/src/Brig/API/Public.hs

```diff
@@ -1134,11 +1141,23 @@ listConnections uid req = do
           -- Is this type actually useless? Or the tests not good enough?
           mtpPagingState = Public.ConnectionPagingState table (LBS.toStrict . C.unPagingState <$> pwsState)
         }
+
     mkState :: ByteString -> C.PagingState
     mkState = C.PagingState . LBS.fromStrict
 
-    convertedState :: Maybe C.PagingState
-    convertedState = fmap mkState . Public.mtpsState =<< Public.gmtprState req
+    localsAndRemotes :: Local UserId -> Maybe C.PagingState -> Range 1 500 Int32 -> Handler Public.ConnectionsPage
+    localsAndRemotes self pagingState size = do
+      localPage <- pageToConnectionsPage Public.PagingLocals <$> Data.lookupLocalConnectionsPage self pagingState (rcast size)
+      let remainingSize = fromRange size - fromIntegral (length (Public.mtpResults localPage))
+      if Public.mtpHasMore localPage || remainingSize <= 0
+        then pure localPage {Public.mtpHasMore = True} -- We haven't check the remotes yet, so has_more must always be True here.
```

Comment:
Typo: check → checked. Also in the original code this was pasted from.

Reply:
Thanks! Fixed in d2d0507.